docs/paper/admin/reference/configuration/spigot-configuration.mdx

```diff
@@ -0,0 +1,563 @@
+    seed-jungle:
+      default: '14357619'
+      description: 'The per-structure seed for jungle structures.'
+    seed-mansion:
+      default: '10387319'
+      description: 'The per-structure seed for mansions.'
+    seed-mineshaft:
+      default: 'default'
+      description: 'The per-structure seed for mineshafts.'
+    seed-monument:
+      default: '10387313'
+      description: 'The per-structure seed for monuments.'
+    seed-nether:
+      default: '30084232'
+      description: 'The per-structure seed for the nether structures.'
+    seed-ocean:
+      default: '14357621'
+      description: 'The per-structure seed for the ocean structures.'
+    seed-outpost:
+      default: '165745296'
+      description: 'The per-structure seed for outposts.'
+    seed-portal:
+      default: '34222645'
+      description: 'The per-structure seed for portals.'
+    seed-shipwreck:
+      default: '165745295'
+      description: 'The per-structure seed for shipwrecks.'
+    seed-slime:
+      default: '987234911'
+      description: 'The per-structure seed for slime chunks.'
+    seed-stronghold:
+      default: 'default'
+      description: 'The per-structure seed for strongholds.'
+    seed-swamp:
+      default: '14357620'
+      description: 'The per-structure seed for swamp structures.'
+    seed-trailruins:
+      default: '83469867'
+      description: 'The per-structure seed for trail ruins.'
+    seed-village:
+      default: '10387312'
+      description: 'The per-structure seed for villages.'
+    simulation-distance:
+      default: 'default'
+      description: 'Overrides the simulation distance. Set to -1 or "default" to use the value in server.properties.'
+    thunder-chance:
+      default: '100000'
+      description: 'The chance of lightning occurring during a thunderstorm,
+        as a probability of 1/<thunder-chance> per chunk, every tick.'
+    ticks-per:
+      hopper-check:
+        default: '1'
+        description: 'The ticks between checks to pull items.'
+      hopper-transfer:
+        default: '8'
+        description: 'The ticks between hopper item movements.'
+    trident-despawn-rate:
+      default: '1200'
+      description: 'The number of ticks before a trident despawns.'
+    verbose:
+      default: 'false'
+      description: 'Whether to log world settings on startup.'
+    view-distance:
+      default: 'default'
+      description: 'Overrides the view distance.
+        Set to -1 or "default" to use the value in server.properties'
+    wither-spawn-sound-radius:
+      default: '0'
+      description: 'The number of blocks that the wither spawn sound is audible.
+        Set to 0 for global (including cross-dimension).'
+    zombie-aggressive-towards-villager:
+      default: 'true'
+      description: 'Whether zombies try to seek out and attack villagers.'
+config-version:
+  default: '12'
+  description: 'Internal constant for upgrading configuration: **Do Not Touch**.'
```

Comment:
I don't think this should be documented or the paper config must also contains this information

Reply:
i was going off of being a complete reference, from what was generated. Im viewing it from the perspective of someone who has no idea what it might mean, so having somewhere to tell them to leave it alone is beneficial. (probably for paper configs as well)